src/OpenSpaceToolkit/Astrodynamics/Dynamics/Thruster/GuidanceLaw/ConstantThrust.cpp

```diff
@@ -0,0 +1,98 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Core/Error.hpp>
+#include <OpenSpaceToolkit/Core/Utilities.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Geometry/3D/Transformations/Rotations/Quaternion.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/Dynamics/Thruster/GuidanceLaw/ConstantThrust.hpp>
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/LocalOrbitalFrameFactory.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace dynamics
+{
+namespace thruster
+{
+namespace guidancelaw
+{
+
+using ostk::math::geom::d3::trf::rot::Quaternion;
+
+using ostk::astro::trajectory::LocalOrbitalFrameFactory;
+using ostk::astro::trajectory::LocalOrbitalFrameDirection;
+
+ConstantThrust::ConstantThrust(const LocalOrbitalFrameDirection& aThrustDirection)
+    : GuidanceLaw("Constant Thrust"),
+      localOrbitalFrameDirection_(aThrustDirection)
+{
+}
+
+ConstantThrust::~ConstantThrust() {}
+
+std::ostream& operator<<(std::ostream& anOutputStream, const ConstantThrust& aConstantThrust)
+{
+    aConstantThrust.print(anOutputStream);
+
+    return anOutputStream;
+}
+
+LocalOrbitalFrameDirection ConstantThrust::getLocalThrustDirection() const
+{
+    if (!localOrbitalFrameDirection_.isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("Local Thrust Direction");
+    }
+
+    return localOrbitalFrameDirection_;
+}
+
+Vector3d ConstantThrust::computeAcceleration(
```

Comment:
```suggestion
Vector3d ConstantThrust::computeThrustAccelerationAt(
```
to make it more explicit?